worker-macros/src/durable_object.rs

```diff
@@ -109,24 +124,123 @@ pub fn expand_macro(tokens: TokenStream) -> syn::Result<TokenStream> {
 
                             #method
                         }
-                    }
+                    },
+                    "on_message" => {
```

Comment:
I would prefer to see this keep the same name that's used in the [Cloudflare documentation](https://developers.cloudflare.com/durable-objects/api/websockets/). So I think this fn should be called `websocket_message`.